app_legacy/Helpers/render/achievement.php

```diff
@@ -69,16 +69,18 @@ function renderAchievementTitle(?string $title, bool $tags = true): string
     if (!$title) {
         return '';
     }
-
     if (!Str::contains($title, '[m]')) {
         return $title;
     }
-    $span = '';
+
+    $missableTag = '';
     if ($tags) {
-        $span = '<span class=\'tag missable\' title=\'Missable\'><abbr>[<b>m</b>]</abbr></span>';
+        $missableTag = "<span class='tag missable' title='Missable'><abbr>[<span>m</span>]</abbr></span>";
     }
+    $title = str_replace('[m]', '', $title);
+    $title = str_replace('  ', ' ', $title);
```

Comment:
Why is this necessary? The browser should automatically collapse whitespace for us, so we can offload that processing to the client.

Does this also collapse three or more spaces? My local DB only has ~500/280k achievements with at least two spaces next to each other, 90 with at least three, and 15 with four!

Reply:
Good point. I think my intention was merely to collapse the extra spaces resulting from misplaced `[m]`s. But indeed, browsers ignore any extra, preceding or succeeding whitespace, so we don't have to bother.